src/node_url.cc

```diff
@@ -1315,8 +1317,10 @@ namespace url {
     CHECK(args[4]->IsFunction());
     Utf8Value input(env->isolate(), args[0]);
     enum url_parse_state override = kUnknownState;
-    if (args[1]->IsNumber())
-      override = (enum url_parse_state)(args[1]->Uint32Value());
+    if (args[1]->IsNumber()) {
+      override = (enum url_parse_state)(
+          args[1]->Uint32Value(env->context()).FromJust());
```

Comment:
Can you turn this into a `reinterpret_cast` and rename `override` to something that’s not a language keyword?